### PR TITLE
Add skater recipe

### PR DESCRIPTION
**Pull request by benvandyke**
Add a recipe for the Skater model interpretation package. 

https://github.com/datascienceinc/Skater/
https://pypi.python.org/pypi/skater

### CODE REVIEW

File: recipes/skater/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{% set name = "skater" %}
+{% set version = "1.0.0b10" %}
+{% set md5 = "39c5c7b616badf6f84d437d7c56936ce" %}
```
Comment (synapticarbors) [inline recipes/skater/meta.yaml]: Please use the `sha256` hash: `96c5b9ce7f32adc066703d5f6c30dfdd0d30dd990e8bc0538c3feb8d10e2c087`

File: recipes/skater/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
```
Comment (synapticarbors) [inline recipes/skater/meta.yaml]: It looks like only `python` and `setuptools` are required to build the package.

File: recipes/skater/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    - scipy >=0.15
+    - numpy >=1.10
+    - scikit-learn >=0.18
+    - pandas >=0.19
+    - pathos
+    - dill
+    - requests
+  run:
+    - python
+    - cython
```
Comment (synapticarbors) [inline recipes/skater/meta.yaml]: Is `cython` required?

File: recipes/skater/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    - scipy >=0.15
+    - numpy >=1.10
+    - scikit-learn >=0.18
+    - pandas >=0.19
+    - pathos
+    - dill
+    - requests
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/skater/meta.yaml]: It looks like `ds-lime` is a run time requirement, but it's missing here.

File: recipes/skater/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    - scikit-learn >=0.18
+    - pandas >=0.19
+    - matplotlib
+    - pathos
+    - dill
+    - requests
+
+test:
+  imports:
+    - skater
```
Comment (synapticarbors) [inline recipes/skater/meta.yaml]: Please add imports of the submodules as well:

```
- skater.model
- skater.data
```

File: recipes/skater/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+about:
+  home: https://github.com/datascienceinc/Skater
+  license: MIT
+  license_family: MIT
```
Comment (synapticarbors) [inline recipes/skater/meta.yaml]: Please add the `license_file` as in:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually

and then put in an issue or pull request upstream to have the license file added to the `MANIFEST.in`.

File: recipes/skater/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    models; as long as you can obtain inputs, and use a function to obtain
+    outputs, you can use Skater to learn about the models internal
+    decision policies.
+  doc_url: https://datascienceinc.github.io/Skater/index.html
+  dev_url: https://github.com/datascienceinc/Skater
+
+extra:
+  recipe-maintainers:
+    - benvandyke
+    - aikramer2
```
Comment (synapticarbors) [inline recipes/skater/meta.yaml]: @aikramer2 -- are you ok with being added as a co-maintainer of the `skater` recipe on conda-forge (you were added by @benvandyke)?

Comment (aikramer2) [inline recipes/skater/meta.yaml]: sure, thanks

**Comment by benvandyke** [comment]
@synapticarbors Thanks for your review. Let's put this one on hold, I need to get the `ds-lime` dependency in first.

**Comment by synapticarbors** [comment]
@benvandyke -- Ok, when you put in the PR for `ds-lime`, ping me and I'll try to review it as soon as I can.

**Comment by benvandyke** [comment]
@synapticarbors The appveyor build is failing with this error message:
```
The following specifications were found to be in conflict:
  - python 3.6*
  - skater 1.0.0b11 py36_0
Use "conda info <package>" to see the dependencies for each package.
```

I built this on my local machine just fine with a py3.6 env. Any ideas?

**Comment by benvandyke** [comment]
@synapticarbors Any thoughts on my issue with Appveyor? I'm having a similar problem with Windows builds on an R recipe. Are there any docs or hints on getting builds to work on Windows? I don't have a Windows machine to test. Maybe try spinning up a Windows VM?

**Comment by synapticarbors** [comment]
I think the error is due to `pathos` missing python 3.6 binaries on windows. See:

https://anaconda.org/conda-forge/pathos/files

The error isn't immediately clear from the appveyor error logs and I'm not sure why those binaries are missing. I just opened an issue here: https://github.com/conda-forge/pathos-feedstock/issues/5

**Comment by synapticarbors** [comment]
Looks like the windows binaries are now up. Could you try initiating another build? I think you can just close and then immediately re-open this PR.

**Comment by synapticarbors** [comment]
@benvandyke Looks like all of the window builds are now passing. As soon as Travis finishes up, I'll merge this.

**Comment by benvandyke** [comment]
@synapticarbors Thanks for your help. Looking forward to contributing more to the conda-forge community. Appreciate your patience on this.

**Pull request merged**